File: src/Application Specific/Classes/Entity Collections/CostCentres/LinkTypes/LinkTypeSql.vb
Imports FSM.Entity.Sys

Namespace Entity.CostCentres.LinkTypes

    Public Class LinkTypeSql
        Private _database As Database

        Public Sub New(ByVal database As Database)
            _database = database
        End Sub

        Public Function [GetAll]() As DataView
            _database.ClearParameter()
            Return New DataView(_database.ExecuteSP_DataTable("CostCentreLinkType_GetAll"))
        End Function

    End Class

End Namespace

File: apollo/apollo.fsm/src/Application Specific/Classes/Entity Collections/CostCentres/LinkTypes/LinkTypeSql.cs
using System.Data;
using FSM.Entity.Sys;

namespace FSM.Entity.CostCentres.LinkTypes
{
    public class LinkTypeSql
    {
        private Database _database;

        public LinkTypeSql(Database database)
        {
            _database = database;
        }

        public DataView GetAll()
        {
            _database.ClearParameter();
            return new DataView(_database.ExecuteSP_DataTable("CostCentreLinkType_GetAll"));
        }
    }
}